### DISCUSSION TITLE
how to access mitmweb by local area ip

### DISCUSSION THREAD

**Discussion by Stephen0910** [mitmproxy/mitmproxy#6319] [Q&A] [answered]
After the deployment is complete in linux， how to access the mitmweb url by other computer(windows).

**Comment by mhils** [comment, marked as answer]
mitmweb's web interface listens on 127.0.0.1 only by default and is not
reachable from other machines in the local network. You can change this by
modifying the web_host option, for example by setting it to "0.0.0.0" to
listen on all interfaces.

**Reply by TiloGit** [reply]
example: 

`sudo mitmweb -p 8881 --web-port 8882 --set web_host=0.0.0.0`

**Comment by Stephen0910** [comment]
follow your suggestion, the web can be aceessed in the same computer, but the different ip was rejected.
The web can not always running in windows, but linux is good.
so I really need the feature. That's just a recommendation.
Thank you for your patience.

**Comment by Stephen0910** [comment]
it works, thanks.
